distributed/tests/test_worker.py

```diff
@@ -311,7 +311,7 @@ async def test_worker_port_range(s):
 
 
 @pytest.mark.slow
```

Comment:
I would expect this test run very fast, assuming the timeout in wait_for is reduced. At least after the report is turned off.

Reply:
It tests that the worker waits for at least three seconds.  I though about removing it but I think that it genuinely adds signal.